components/ws-daemon-api/typescript/package.json

```diff
@@ -14,15 +14,16 @@
   "dependencies": {
     "@gitpod/content-service": "0.1.5",
     "@grpc/grpc-js": "^1.3.7",
-    "google-protobuf": "^3.17.3",
+    "google-protobuf": "^3.18.0-rc.2",
```

Comment:
Why move to 3.18.0-rc.2 when there's [3.19.1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.19.1)?
The latter is not a pre-release and also fixed some Javascript issue

Reply:
Because that has been release after this upgrade has been performed. NPM ecosystem moving faaaast :fast_forward: :see_no_evil:

Reply:
We could go with [3.18.0](https://github.com/protocolbuffers/protobuf/releases/tag/v3.18.0) which looks a bit more mature then 3.19. And has no big diff wrt JS.

But honestly I think it's better to do this after the merge to get this thing out of the way. :slightly_smiling_face:

Reply:
`3.19+3.19.1` JS fixes are mostly relevant for JS in web browsers. 

please, let's move on to have enough time for testing on gitpod-staging.com.

Reply:
3.19.1 is being updated in https://github.com/gitpod-io/gitpod/pull/6461 (across all components)